src/views/GovernanceReviewTeam/GRBReview/PresentationLinksCard.tsx

```diff
@@ -84,6 +87,35 @@ function PresentationLinksCard({
       });
   };
 
+  // External link modal handling
```

Comment:
This handling block is copied from the RTE feature. It isn't great .. but if it needs iteration I think the next worthy step would be to generalize the hook and effect for the whole app, instead of placing a selector scope made from assigning dom refs. (At the time didn't think the site would have so many external links.) For now I think this duplicated instance is okay.

Reply:
#2970 introduces an `ExternalLinkAndModal` component that can be used here. We can update this later, just wanted to point it out for visibility.

Reply:
I had referenced Gary to this as he started that work. Thanks.